packages/store/test/StoreCore.t.sol

```diff
@@ -80,21 +41,18 @@ contract StoreCoreTest is Test, StoreReadWithStubs {
     vm.expectEmit(true, true, true, true);
     emit StoreSetRecord(StoreCoreInternal.SCHEMA_TABLE, key, abi.encodePacked(schema.unwrap(), keySchema.unwrap()));
 
-    // !gasreport StoreCore: register schema
-    StoreCore.registerSchema(table, schema, keySchema);
+    IStore(this).registerSchema(table, schema, keySchema);
```

Comment:
We don't have to cast to `IStore` here since `this` already extends the `StoreMock` contract, but I figured it makes it a bit more explicit to see what's going on here (and we don't care about gas cost in here because gas reports are moved to `StoreCoreGas`)